rules/codelingo/effective-go/comment-first-word-as-subject/codelingo.yaml

```diff
@@ -291,8 +291,21 @@ tenets:
       codelingo/docs:
         title: Comment First Word as Subject
         body: |
-          Doc comments work best as complete sentences, which allow a wide variety of automated presentations.
+          Doc comments work best as complete sentences to allow a wide variety of automated presentations.
           The first sentence should be a summary that starts with the name being declared.
+
+          ## Bad code
+          ```
+          /* This func comment should begin with 'bar'
```

Comment:
It's more idiomatic in go to use // style comments, especially for doc comments. (I'm not even sure /* */ style would be respected by godoc). Please change these comments to that style. Also please captalise the func names (which means they are exported, which is what this rule is actually checking for)